modules/claims_api/app/workers/claims_api/vmbs_updater.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'bgs'
+
+module ClaimsApi
+  class # frozen_string_literal: true
+
+    require 'sidekiq'
+    require 'bgs'
+    
+    module ClaimsApi
+      class VBMSUpdater
+        include Sidekiq::Worker
+    
+        def perform(power_of_attorney_id)
+          poa_form = ClaimsApi::PowerOfAttorney.find(power_of_attorney_id)
+          service = BGS::Services.new(
+            external_uid: poa_form.external_uid,
+            external_key: poa_form.external_key
+          )
+          service.corporate_update.update_poa_access(
+            participant_id: target_veteran.participant_id,
```

Comment:
i dont think target_veteran is accessible within a background job context?

Reply:
I think you're right, passing into the method instead

Reply:
im pretty sure this won't work either, because of all the issues we've had around sidekiq. Sidekiq only allows super simple arguments.

Reply:
it'll most likely have to be accessed through the database record some way. stored with the auth_headers perhaps.